rust/repo/{{ cookiecutter.repo_name }}/.github/workflows/ci.yaml

```diff
@@ -17,6 +17,10 @@ jobs:
     runs-on: ubuntu-22.04
     steps:
       - uses: actions/checkout@v3
+{%- if cookiecutter.submodules %}
+        with:
+          submodules: true
+{%- endif %}
```

Comment:
actually, i find that checking out submodules when linting is a bad idea, otherwise you have to make everything ignore the submodules, which is tricky